pkgs/applications/version-management/forgejo/default.nix

```diff
@@ -66,46 +70,61 @@ buildGoModule rec {
       --prefix PATH : ${lib.makeBinPath [ bash git gzip openssh ]}
   '';
 
-  passthru.tests = nixosTests.forgejo;
-
-  passthru.updateScript = lib.getExe (writeShellApplication {
-    name = "update-forgejo";
-    runtimeInputs = [
-      common-updater-scripts
-      coreutils
-      curl
-      jq
-      nix
-    ];
-    text = ''
-      releases=$(curl "https://codeberg.org/api/v1/repos/forgejo/forgejo/releases?draft=false&pre-release=false&limit=1" \
-        --silent \
-        --header "accept: application/json")
-
-      stable=$(jq '.[0]
-        | .tag_name[1:] as $version
-        | ("forgejo-src-\($version).tar.gz") as $filename
-        | { $version, html_url } + (.assets | map(select(.name | startswith($filename)) | {(.name | split(".") | last): .browser_download_url}) | add)' \
-        <<< "$releases")
-
-      archive_url=$(jq -r .gz <<< "$stable")
-      checksum_url=$(jq -r .sha256 <<< "$stable")
-      release_url=$(jq -r .html_url <<< "$stable")
-      version=$(jq -r .version <<< "$stable")
-
-      if [[ "${version}" = "$version" ]]; then
-        echo "No new version found (already at $version)"
-        exit 0
-      fi
-
-      echo "Release: $release_url"
-
-      sha256=$(curl "$checksum_url" --silent | cut --delimiter " " --fields 1)
-      sri_hash=$(nix hash to-sri --type sha256 "$sha256")
-
-      update-source-version "${pname}" "$version" "$sri_hash" "$archive_url"
+  passthru = {
+    data-compressed = runCommand "data-compressed" {
+      nativeBuildInputs = [ brotli xorg.lndir ];
+    } ''
+      mkdir $out
+      lndir ${forgejo.data}/ $out/
+
+      # Create static gzip and brotli files
+      find -L $out -type f -regextype posix-extended -iregex '.*\.(css|html|js|svg|ttf|txt)' | while read file; do
```

Comment:
Why not use `find ... -exec gzip --best --keep {} ';' -exec brotli --best --keep {} ';' ` instead of the loop?  (or even `+` instead of `';'` to reduce the amount of invocations).

Reply:
Build error with this variant:
```
find -L $out -type f -regextype posix-extended -iregex '.*\.(css|html|js|svg|ttf|txt)' \
  -exec gzip --best --keep --force {} ';' \
  -exec brotli --best --keep --no-copy-stat {}
```
```
error: builder for '/nix/store/sjain6jfba0ipjwamx15a63x3460wlz1-data-compressed.drv' failed with exit code 1;
       last 10 log lines:
       > /nix/store/b2gjap9zxk3cfcys43caa4hns85gf1nq-forgejo-1.18.5-0-data/templates/repo/find:
       > /nix/store/b2gjap9zxk3cfcys43caa4hns85gf1nq-forgejo-1.18.5-0-data/templates/repo/issue:
       > /nix/store/b2gjap9zxk3cfcys43caa4hns85gf1nq-forgejo-1.18.5-0-data/templates/repo/issue/view_content:
       > /nix/store/b2gjap9zxk3cfcys43caa4hns85gf1nq-forgejo-1.18.5-0-data/templates/repo/issue/fields:
       > /nix/store/b2gjap9zxk3cfcys43caa4hns85gf1nq-forgejo-1.18.5-0-data/templates/repo/issue/labels:
       > /nix/store/b2gjap9zxk3cfcys43caa4hns85gf1nq-forgejo-1.18.5-0-data/templates/repo/graph:
       > /nix/store/b2gjap9zxk3cfcys43caa4hns85gf1nq-forgejo-1.18.5-0-data/templates/repo/migrate:
       > /nix/store/b2gjap9zxk3cfcys43caa4hns85gf1nq-forgejo-1.18.5-0-data/templates/explore:
       > /nix/store/b2gjap9zxk3cfcys43caa4hns85gf1nq-forgejo-1.18.5-0-data/templates/base:
       > find: missing argument to `-exec'
       For full logs, run 'nix log /nix/store/sjain6jfba0ipjwamx15a63x3460wlz1-data-compressed.drv'.

```

Reply:
Fixed.